### ISSUE TITLE
Feature request: Master/detail modes for multiple widgets of the same type on same page (ContentViewDisplayMode)

### ISSUE THREAD

**Issue by Masahigo** [Sitefinity/feather-widgets#1408]
Hi,

I noticed the current Feather widgets do not support this very useful feature in the Web forms side where you can define two widgets (eg. News widgets) to work together on the same page in different modes so that you can list the items in one column and display only the selected detail view in the other.

This was set using this property called ContentViewDisplayMode in the widget's Advanced settings.

This feature would come in very handy at least for News widget and widgets created using Module builder ("Dynamic" widgets) for starters.

**Comment by MerrittMelker** [comment]
I agree but this shouldn't be marked as a feature request: it's either a bug or a missing feature. You can't have 2 new widgets 1 page. It just doesn't work.

**Comment by ElenaGaneva** [comment]
Hi,

Just an update that this functionality is already available with Sitefinity 12.0.7000.0. You can check our [release notes](https://github.com/Sitefinity/feather/releases) for further reference.

Regards,
Elena
